request_logger.go

```diff
@@ -3,10 +3,10 @@ package buffalo
 import (
 	"time"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?